client/src/pages/ExploreCourses.jsx

```diff
@@ -1,91 +1,137 @@
 import React, { useContext, useEffect, useState } from 'react';
 
-import { Button, Container, Item } from 'semantic-ui-react';
+import { Button, Container, Item, Modal } from 'semantic-ui-react';
 import { useLocation } from 'react-router-dom';
 
 import { AuthenticationContext } from '../context/authentication';
 
 import useAccountOperations from '../hooks/accountOperations';
 import useContentManagement from '../hooks/contentManagement';
 
-import CourseList from '../components/CourseList';
-
 import { GetCoursesByCategoryRequest } from '../proto/content-management_pb';
 import { EnrollCourseRequest } from '../proto/account-operations_pb';
 
+import ConfirmationModalBody from '../components/ConfirmationModalBody';
+import CourseList from '../components/CourseList';
+
 import courseImage from '../images/tmp/course.png';
 
 const Courses = props => {
     const [courses, setCourses] = useState([]);
+    const [isModalShown, setIsModalShown] = useState(false);
+    const [selectedCourse, setSelectedCourse] = useState(null);
 
     const { user } = useContext(AuthenticationContext);
 
     const {
-        methods: { getCoursesByCategory },
-        state
+        state: contentManagementState,
+        methods: contentManagementMethods
     } = useContentManagement();
-    const { response } = state;
 
-    const { methods } = useAccountOperations();
-    const { enrollCourse } = methods;
+    const {
+        state: accountOperationsState,
+        methods: accountOperationsMethods
+    } = useAccountOperations();
 
     const location = useLocation();
 
+    const { getCoursesByCategory } = contentManagementMethods;
     useEffect(() => {
         const category = new URLSearchParams(location.search).get('category');
         const request = new GetCoursesByCategoryRequest();
         request.setId(category);
         getCoursesByCategory(request);
     }, [getCoursesByCategory, location]);
 
+    const { response: contentManagementResponse } = contentManagementState;
     useEffect(() => {
-        if (response) {
-            setCourses(response.getCoursesList());
+        if (contentManagementResponse) {
```

Comment:
is there are difference or just code styling?

Reply:
This file uses two types of responses. I just rename it for better understanding